countdown.rb

```diff
@@ -5,64 +5,115 @@
 
 require_relative 'helpers'
 
-clear
 
+a = Artii::Base.new :font => 'ogre'
+b = Artii::Base.new :font => 'catwalk'
+c = Artii::Base.new :font => 'cosmic'
+d = Artii::Base.new :font => 'invita'
+clear
 puts 'Introduce el tiempo en minutos'
 minutes = gets.chomp.to_i
 
 clear
 
 puts 'Introduce la clave de desactivación'
 bomb_key = gets.chomp
-
+accumulated_key = []
+(bomb_key.length).times { accumulated_key << "*" }
 bomb_deactivated = false
 
 final = Time.now + minutes * 60
 
 loop do
-  accumulated_key = []
+  
   pressed_key = nil  
+  count = 0
   begin
     clear
-    puts "SE ESTÁ ACABANDO EL TIEMPO..."
-    puts "#{format_time(time_remaining(final))}"
-
-    puts "Introduce la clave:\n\n"
-
-    print "#{accumulated_key.join}"
+    3.times { puts }
+    puts a.asciify("SE ESTA ACABANDO EL TIEMPO...")
+    3.times { puts }
+    puts d.asciify("#{format_time(time_remaining(final))}")
+    3.times { puts }
+    puts c.asciify("Introduce   la     clave   :\n\n")
+    3.times { puts }
+    print b.asciify("#{accumulated_key.join}")
+    break if Time.now > final
 
     Timeout.timeout(1) do
       pressed_key = STDIN.getch
       raise Timeout::Error
-    end
-
-    break if Time.now > final
+    end 
 
-  rescue Timeout::Error
-    if pressed_key
-        accumulated_key << pressed_key
+    rescue Timeout::Error
+      if pressed_key
+        accumulated_key[count] = pressed_key
```

Comment:
I don't really get what you are doing here. It seems like you are using a counter to indicate the number of chars that have been inserted, and then compare if the introduced character is correct or not. 

If that is the case, I think it would be more explicit to do something like `bomb_key.start_with? accumulated_key` - that will say if the accumulated key is still correct, or if a character has been typed wrong.